server/models/Notifications.js

```diff
@@ -0,0 +1,37 @@
+const mongoose = require('mongoose');
+
+const notificationSchema = new mongoose.Schema({
+    type: {
+        required: true,
+        type: String
+    },
+    createdAt: {
+        required: true,
+        type: Date,
+        default: Date.now
+    },
+    sender: {
+        type: String,
+        required: true,
+    },
+    receiver: {
+        type: String,
+        required: true,
+    },
```

Comment:
What is the intended value to be stored here? If it's supposed to be a reference to a model then I would utilize [Mongoose reference](https://mongoosejs.com/docs/populate.html)

Reply:
receiver and sender will be username or id of the User.